Samples/Client/WebClient/index.html

```diff
@@ -129,11 +213,96 @@
         console.log("setLocalDescription failed");
     }
 
-    function aic_success_cb() {
+    function navOnMouseDown(event)
+    {
+        isMouseDown = true;
+
+        mouseDownX = event.clientX;
+        mouseDownY = event.clientY;
+
+        downPitch = navPitch;
+        downHeading = navHeading;
+        downLocation[0] = navLocation[0];
+        downLocation[1] = navLocation[1];
+        downLocation[2] = navLocation[2];
+
+        preventDefault();
```

Comment:
do you mean `event.preventDefault()` ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault

Reply:
yes, this should be event.preventDefault()